raiden/tests/utils/blockchain.py

```diff
@@ -209,7 +209,8 @@ def geth_create_blockchain(
         p2p_ports,
         base_datadir,
         verbosity,
-        genesis_path=None):
+        genesis_path=None,
```

Comment:
Can you maybe add a docstring to this function into which you also provide the example usage gist you linked in the PR's comments themselves?